### PR TITLE
Update glossary to add foreign national terms

### LINKED ISSUE #5042: Add new glossary terms related to foreign nationals and update "Cooperatives" term
### Summary

OGC has approved new glossary definitions for "lawfully admitted for permanent residence" and "U.S. National." These terms, in addition to making the term "Cooperatives" singular, need to be included for an update to the glossary.

### Related issues

- #4969 
- #4818 

### Completion criteria

- [x] Add PRs to the appropriate branches
- [x] Test in staging after release is cut
- [x] Link glossary items to "Foreign National page" once everything is on production

### PR DESCRIPTION
**Pull request by djgarr**
## Summary (required)

Adds two terms to glossary

https://docs.google.com/spreadsheets/d/1iSHhncHt8JCzYrh_UEHmyhoy3sKEpAnNNspCYdS188g/edit#gid=0 (see highlighted terms)

- #5042

### CODE REVIEW

File: fec/fec/static/js/data/terms.json [hunk 1]
```diff
@@ -318,6 +318,10 @@
   {
     "term": "Labor organization",
     "definition": "An organization, agency or employee representative committee or plan, in which employees participate and which exists for the purpose of dealing with employers on grievances, labor disputes, wages, hours of employment or working conditions. 11 CFR <a href=\"https://www.fec.gov/regulations/114-1/CURRENT#114-1-d\">114.1(d)</a>."
+  },
+    {
+    "term": "Lawfully admitted for permanent residence",
+    "definition": "An individual lawfully admitted for permanent residence is someone who has been granted authorization to live and work in the United States on a permanent basis. As proof of that status, U.S. Citizenship and Immigration Services (USCIS) grants a person a permanent resident card, commonly called a "Green Card.""
```
Comment (patphongs) [inline fec/fec/static/js/data/terms.json:324]: @djgarr When there are quotations as part of the text, it needs to be escaped with a backwards slash. Like this `...called a \"Green Card.\""`. After that change, it'll work, thanks!

**Review by patphongs — CHANGES_REQUESTED** [review]

File: fec/fec/static/js/data/terms.json (hunk 1 above)
Comment (djgarr) [reply fec/fec/static/js/data/terms.json]: Text updated

**Review by dorothyyeager — APPROVED** [review]
Approved from content team perspective. Thanks David, nice job!

**Review by patphongs — APPROVED** [review]
Looks good, thanks @djgarr !

Screen Shot 2022-01-28 at 9 22 48 AM

**Pull request merged**